pages/common/sl.md

```diff
@@ -13,3 +13,11 @@
 - Let the train fly:
 
 `sl -F`
+
+- Make the train small:
+
+`sl -l`
```

Comment:
How is `-l` related to small ? Is it possible to use a word starting with `l` ?

Reply:
As per the man page `-l` is for Little.

Reply:
Cool, then let's change it to `Make the train little`.